src/MemObject.h

```diff
@@ -168,19 +168,17 @@ class MemObject
 
     SwapOut swapout;
 
-    /* TODO: Remove this change-minimizing hack */
-    using Io = Store::IoStatus;
-    static constexpr Io ioUndecided = Store::ioUndecided;
-    static constexpr Io ioReading = Store::ioReading;
-    static constexpr Io ioWriting = Store::ioWriting;
-    static constexpr Io ioDone = Store::ioDone;
+    static constexpr Store::IoStatus ioUndecided = Store::ioUndecided;
+    static constexpr Store::IoStatus ioReading = Store::ioReading;
+    static constexpr Store::IoStatus ioWriting = Store::ioWriting;
+    static constexpr Store::IoStatus ioDone = Store::ioDone;
```

Comment:
>  TODO: Remove this change-minimizing hack
   
Here, "this hack" includes `Io` declaration _and_ four static members that use that `Io` declaration. Please remove _all_ of them and adjust the affected code to use Store::IoStatus directly, avoiding any shims.

```suggestion
```

Reply:
removed these 4 lines and replaced with Store::IoStatus wherever necessary